components/ContactInfo.vue

```diff
@@ -3,18 +3,30 @@
 		<h2>
 			Contact Us
 		</h2>
-		<p>
+		<div>
 			<a :href="'mailto:' + email">{{ email }}</a><br>
 			IDRC<br>
 			205 Richmond Street West<br>
-			<img src="~/static/img/idrc.png">
-		</p>
+			<div class="logo-ocadu-wrapper">
+				<IdrcLogo />
+				<div class="logo-ocadu">
+					<OcaduLogo />
```

Comment:
Should these be linked to the corresponding websites? Whether or not we go that route, they need accessible labels.